sycl/test-e2e/Matrix/joint_matrix_bfloat16_impl.hpp

```diff
@@ -122,13 +120,21 @@ int main() {
 
     if (combinations[i].nsize == 16) { // architecture::intel_gpu_pvc
       test<bfloat16, float, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      // test<bfloat16, bfloat16, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
 
       // This combination is not currently supported for sub group size = 32 in
       // IGC
 #if (!defined(SG_SZ) || SG_SZ != 32)
       test<bfloat16, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+      // test<bfloat16, bfloat16, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
       test<bfloat16, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
-      test<bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+      // test<bfloat16, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
```

Comment:
AFAIK, it is not supported. Did you try to run the test? Does it pass?

Reply:
No, all these added combinations do not currently pass.
I missed commenting them as well.